nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/PropertyCommentManager.java

```diff
@@ -84,7 +95,7 @@
 
     @Override
     public DocumentModel createComment(DocumentModel docModel, String comment) {
-        return createComment(docModel, comment, docModel.getCoreSession().getPrincipal().getName());
+        throw new UnsupportedOperationException();
```

Comment:
Maybe add a message stating that this new service doesn't implement deprecated API ?